### ISSUE TITLE
Feature Request: Copy all opened tabs to clipboard

### ISSUE THREAD

**Issue by johndoe217** [tuyafeng/Via#945]
When you have hundreds of tabs opened it's impossible to handle properly with mobile browser.
In Chrome you have a feature which let you copy all opened tabs to clipboard, You can then paste them to any of the pastebin sites and then reopen them in desktop browser.
This feature is quite simple and yet solved a very common issue with mobile browsing experience: tabs management.

Thank you for making Via browser and I hope you can add this feature soon.

**Comment by tuyafeng** [comment]
In fact, what I want to know is the content format of the text. For example, is it a string similar to the following:
```
https://www.google.com.hk/search?newwindow=1&q=a
https://www.google.com.hk/search?newwindow=1&q=b
...
```

**Comment by johndoe217** [comment]
> I was surprised by the number of open tabs. Can you give me an example of pastebin site? I want to know the format of the pasted content.

It's a bad habit of mine to open too many tabs intended for later reading and never get to them until eventually the browser crashed and take all my tabs with it. The tabs count are always smiling at me :)

As for the format, Google Chrome use "share group" feature to allow the user to share or copy the URLs of all tabs within a group, the data is exported as:
```
1. https://en.m.wikipedia.org/wiki/Google_chrome
2. https://en.m.wikipedia.org/wiki/Google
3. https://en.m.wikipedia.org/wiki/Web_browser
```
Via don't have group feature so export all opened tab feature is requested.

Lastly for the pastebin site. It don't really matter which site you use or what format because you are just pasting plain text but I highly recommend https://paste-bin.xyz due to 2 reasons:
1. it can be pasted as markdown which will automatically turn all the URLs into clickable links
2. the result page URL is very short compared to other sites which make it easy to memorize and type them on desktop

**Comment by h599** [comment]
Yes, I've experienced this crashing issue too.. many times, I usually opened too many tabs approximately 150 tabs

Please @tuyafeng add this lovely feature, and add feature to export them to a file if possible 🙏 Thank you

And +Please add option to how via browser should react when user open saved mht pages, the previous version did it better 🙏 (the new update and old way to open saved mht pages can be selected via settings page)

Because I usually open those heavy data pages when connected with wifi and saved them to memory, then open them at home when wifi connection isn't availabe, I'm sure this will help many people who are unfortunate with internet connection availability, thank you tufayeng, for creating this features-rich browser available to us

Thank you 🙏

**Comment by tuyafeng** [comment]
@h599 Thanks for your feedback. But I wanna know what dost that mean “can not be selected via settings page”?

**Comment by h599** [comment]
Hi @tuyafeng ,did you meant this line?

"the new update and old way to open saved mht pages can be selected via settings page"

via=on/from 

I meant the newer one can have both option available, and user can choose how via should react when user choose to open via's saved offline pages

**Comment by adan89lion** [comment]
It will certainly be a nice addition to Via Browser. It is possible to implement similarly like Samsung Internet Browser, where you can select multiple tabs to close/group/share. Sharing from Samsung Internet will copy website title and links, so it can be informative if where I paste doesn't support link preview. Other major browsers like Chrome, Firefox and Safari (on iOS) already support copying links from multiple tabs, and it can vastly improve the productive workflow of Via power users.
